card/db/schema.rb

```diff
@@ -66,30 +68,48 @@
   add_index "card_revisions", ["creator_id"], name: "revisions_created_by_index", using: :btree
 
   create_table "cards", force: :cascade do |t|
-    t.string   "name",                limit: 255,      null: false
-    t.string   "key",                 limit: 255,      null: false
-    t.string   "codename",            limit: 255
     t.integer  "left_id",             limit: 4
-    t.integer  "right_id",            limit: 4
-    t.integer  "current_revision_id", limit: 4
     t.datetime "created_at",                           null: false
     t.datetime "updated_at",                           null: false
+    t.integer  "current_revision_id", limit: 4
+    t.string   "name",                limit: 255,      null: false
     t.integer  "creator_id",          limit: 4,        null: false
     t.integer  "updater_id",          limit: 4,        null: false
+    t.integer  "right_id",            limit: 4
+    t.string   "key",                 limit: 255,      null: false
+    t.boolean  "trash",                                null: false
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.